components/server/src/api/organization-service-api.ts

```diff
@@ -283,6 +283,14 @@ export class OrganizationServiceAPI implements ServiceImpl<typeof OrganizationSe
                     throw new ApplicationError(ErrorCodes.BAD_REQUEST, "invalid defaultRole");
             }
         }
+        if (typeof req.timeoutSettings?.allowChangeByMembers === "boolean") {
+            update.timeoutSettings = {
+                allowChangeByMembers: req.timeoutSettings.allowChangeByMembers,
+            };
+            if (typeof req.timeoutSettings.inactivity === "object") {
```

Comment:
Good catch, they shouldn't be. Originally, I did this because of the shape of `TimeoutSettings` which always required `allowChangeByMembers`, but that did not make much sense and therefore we can change it back.